OracleGoldenGate/21c/README.md

```diff
@@ -32,11 +37,33 @@ For more information about Oracle GoldenGate please see the [Oracle GoldenGate 2
 ## Build an Oracle GoldenGate Container Image
 
 Once you have downloaded the Oracle GoldenGate software, a container image can be created using the Docker command-line interface.
-A single `--build-arg` is needed to indicate the GoldenGate installer which was downloaded.
+Two arguments are required to build the container image. 
+A `--build-arg` `INSTALLER` is needed to indicate the GoldenGate installer which was downloaded. 
+Another `--build-arg` `INSTALL_OPTION` to specify the GoldenGate database type.
 
-```sh
+Please refer to the following table for more information:
+
+| GoldenGate deployment     | INSTALL_OPTION |
+|:--------------------------|:---------------|
+| GoldenGate for Oracle 20c | `ORA20c`       |
+| GoldenGate for BigData    | `Generic`      |
+| GoldenGate for MySQL      | `MySQL`        |
+
+
+To create a docker image for GoldenGate for BigData, use the following script:
+```bash
+#!/bin/bash
+
+docker build --tag=oracle/goldengate-for-bigdata:21.3.0.0.0 \
+                --build-arg INSTALLER=ggs_Linux_x64_BigData_services.21.5.0.0.0_220222.0938.zip \
+                --build-arg INSTALL_OPTION=Generic .
+```
+
+To create a docker image for GoldenGate for Oracle, use the following script:
```

Comment:
Please replace the word "docker" with "container"

Reply:
Done